src/hooks/use-service-worker.ts

```diff
@@ -1,32 +1,65 @@
 import { useEffect, useState } from 'react';
+import { clearInterval } from 'timers';
```

Comment:
what is this? do we really need this? timers is actually a node package or not?

Reply:
ah damn auto-import... of course I wanted to use the browser version and not the node version 🙄